src/fty-discovery.cfg.in

```diff
@@ -13,6 +13,17 @@ discovery
     scans
     ips
     documents
+    defaultValues
+        aux
+            create_mode = "3"
+            create_user = "system"
+            parent = "0"
+            priority = "5"
+            status = "nonactive"
+        links
+            1
```

Comment:
1 is an cfg entry? Could you clarify this?

Reply:
It's for adding multiple power links, if needed. Not that I expect this code will be around after the release since it's way overdue for a complete rewrite.